docs/modules/jenkins-agents/partials/update-to-3x.adoc

```diff
@@ -1,6 +1,6 @@
 == Changes to Jenkins agent images
 
-Some agent images (`jenkins-agent-maven`, `jenkins-agent-nodejs10-angular`, `jenkins-agent-scala`) exposed the environment variables `NEXUS_HOST` and `NEXUS_USERNAME`/`NEXUS_PASSWORD`/`NEXUS_AUTH`. This was not done intentionally, but as a side effect of using those variables during image build time. In 3.x, that has been cleaned up. Now those variables are passed as build arguments to the image build, which means they are no longer accessible as environment variables in the running container. Nexus credentials can and should be accessed from the pipeline context (`context.nexusUrl`, `context.nexusUsername` and `context.nexusPassword`). As a consequence, if a `Jenkinsfile` relies on the presence of those environment variables without setting them explicitly using `withEnv`, it will fail to run in 3.x. That said, the `Jenkinsfile` templates which shipped by default with the quickstarters did not rely on the environment variables - with the exception of `fe-ionic` which did not set `NEXUS_HOST` explicitly in the `Build` stage.
+Some agent images (`jenkins-agent-maven`, `2-angular`, `jenkins-agent-scala`) exposed the environment variables `NEXUS_HOST` and `NEXUS_USERNAME`/`NEXUS_PASSWORD`/`NEXUS_AUTH`. This was not done intentionally, but as a side effect of using those variables during image build time. In 3.x, that has been cleaned up. Now those variables are passed as build arguments to the image build, which means they are no longer accessible as environment variables in the running container. Nexus credentials can and should be accessed from the pipeline context (`context.nexusUrl`, `context.nexusUsername` and `context.nexusPassword`). As a consequence, if a `Jenkinsfile` relies on the presence of those environment variables without setting them explicitly using `withEnv`, it will fail to run in 3.x. That said, the `Jenkinsfile` templates which shipped by default with the quickstarters did not rely on the environment variables - with the exception of `fe-ionic` which did not set `NEXUS_HOST` explicitly in the `Build` stage.
```

Comment:
`2-angular` seems like a mistake. Maybe some replacement gone wrong?

I think this file should not change at all - it's an update from 2.x to 3.x, which should not be affected by new developments.

Reply:
That's true - it was a mistake when replacing all occurrences. Let me put it back.